webapp/src/containers/Rhs/projectDetails/header.tsx

```diff
@@ -0,0 +1,123 @@
+import React, {useState} from 'react';
+import {useDispatch} from 'react-redux';
+
+import plugin_constants from 'plugin_constants';
+
+import BackButton from 'components/buttons/backButton';
+import PrimaryButton from 'components/buttons/primaryButton';
+import ConfirmationModal from 'components/modal/confirmationModal';
+import ToggleSwitch from 'components/toggleSwitch';
+import Dropdown from 'components/dropdown';
```

Comment:
is the import order correct here?

Reply:
there is no hard rule on import order for same type like we can import all `components` together and their order matters but inside that I don't think any rule is set